app/templates/application.hbs

```diff
@@ -36,6 +36,9 @@
                     {{#link-to "collectionobject.new" tagName="li"}}
                         <a href="#">{{t "navigation.collection_object.add"}}</a>
                     {{/link-to}}
+                    {{#link-to "collection" tagName="li"}}
```

Comment:
I think the link to "Samlingar" should be its own item in the navigation bar. See screenshot: 
![screenshot 2016-06-30 14 05 56](https://cloud.githubusercontent.com/assets/186815/16487560/07eb6514-3ecc-11e6-8b5d-a01388e62afa.png)